task_8_9_6.py

```diff
@@ -0,0 +1,13 @@
+from utils import print_intro
+
+print_intro()
+
+
+def main():
+    year = int(input('Enter year: '))
+    print('Leap year.' if year % 4 == 0 and year % 100 != 0 or year % 400 == 0
+          else 'Ordinary year.')
```

Comment:
maybe it would be better to return Leap or Ordinary.
it would be much easier to test